src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final Random random = new Random();
+
+    public Ball getRandomBall() {
+        Color randomColor = ColorSupplier.getRandomColor();
```

Comment:
The `ColorSupplier.getRandomColor()` method is being called in a static context. This goes against the checklist item that advises against using static methods. You should create an instance of `ColorSupplier` and call the `getRandomColor()` method on that instance.